parser.go

```diff
@@ -73,6 +73,8 @@ func (opt *Options) Parse() (err error) {
 	if err != nil {
 		return
 	}
+	defer xmlFile.Close()
```

Comment:
This fixes an unrelated bug. The `defer` keyword does not close the file immediately; rather, it ensures that the file is closed regardless of whether there is an early `return` or a `panic`.